velox/buffer/Buffer.h

```diff
@@ -64,15 +64,15 @@ class Buffer {
   virtual ~Buffer() {}
 
   void addRef() {
-    referenceCount_.fetch_add(1);
+    referenceCount_.fetch_add(1, std::memory_order_relaxed);
```

Comment:
`referenceCount_` is used for mutability check (single referenced buffer is considered mutable).  Reordering is dangerous in this use case.